### ISSUE TITLE
Not all columns are copied to the google sheet

### ISSUE THREAD

**Issue by Laila92** [robin900/gspread-dataframe#13]
I am trying to write my pandas dataframe to a google sheet using the gspread_data package. Adding include_index=True should add an extra column to the sheet with the index from the dataframe, but it does not add it for me. My code:

```
from gspread_dataframe import get_as_dataframe, set_with_dataframe
df=pd.read_csv(r'C:\Users\Laila\Desktop\rawData18Sheet.csv',thousands=',', 
index_col='Client',parse_dates=True,encoding = "ISO-8859-1",usecols=columns)

test_sheet = client.open("2018 Raw Data").worksheet("test_sheet")

set_with_dataframe(test_sheet, df,resize=True, 
include_index=True)
```
I did set the index when I read the dataframe from a csv, and pulling the index with df.index.values gives me the values I want in the index of the google sheet.

**Comment by robin900** [comment]
@Laila92 Thanks for reporting your problem. Would you be able to attach your rawData18Sheet.csv file here? Or perhaps even just the first three lines of it, including the header? I want to reproduce your problem using your own data file's column structure.

**Comment by robin900** [comment]
Also @Laila92 -- if one of your columns in the input CSV file is named `'Client'`, then `include_index=True` merely means not to omit the `Client` column in the written Google worksheet. If instead you use an implicit index column, by omitting `index_col='Client'`, then you'll see an additional column added to the worksheet via `include_index=True`. If this answers your question, hooray; if not, let me know and attach if you can a sample of your input CSV file.

**Comment by Laila92** [comment]
I can get the Client column to appear if I do not set is as an index, but I prefer having that column as the index of my dataframe. Why, if I set `index_col='Client`', does `include_index=True` not make the client column show up in the google sheet?

**Comment by Laila92** [comment]
Client | has service? | year | month | traffic | Source | Leads | Bad Leads | Sold | Gross | new leads | type | conversion rate | closing rate |   |   |   |   |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Example_client | x | 2018 | January | 18978 | DEALER | 355 | 7 | 23 | $16,708.32 | 362 | website | 0.01907471809 | 0.0635359116 |   |   |   |   |   |   |   |   |   |   |   |  
Example_client | x | 2018 | January | 18978 | Lives In Area | 111 | 0 | 48 | $72,054.06 | 111 | other | 0.005848877648 | 0.4324324324 |   |   |   |   |   |   |   |   |   |   |   |  
Example_client | x | 2018 | January | 18978 | Sales | 86 | 0 | 84 | $93,160.23 | 86 | other | 0.004531562862 | 0.976744186 |   |   |   |   |   |   |   |   |   |   |   |

**Comment by robin900** [comment]
@Laila92 Thanks for the extra info; I'll give you an update today (NYC time).

**Comment by robin900** [comment]
@Laila92 I've released to PyPI version 3.0.2 of `gspread-dataframe` with a fix (including test coverage) for your reported problem. Please let me know whether the release works as expected for you. !תודה רבה

**Comment by Laila92** [comment]
It works, thank you so much.
